<?xml version="1.0"?>
<launch>
  <Wait length="20"/>
  <arg name="namespace" default="wamv"/>

  <!-- Keyboard teleop -->
  <node pkg="robotx_2022" type="boat_twist.py" name="boat_twist" output="screen">
  </node>

  <node pkg="robotx_2022" type="wamv_dk_functions.py" name="wamv_dk_functions" output="screen">
    <param name="input_right_cmd" value="1"/>
    <param name="input_left_cmd" value="1"/>
    <param name="input_angle" value="0"/>
    <param name="iter_num" value="0"/>
    <remap from="left_thrust_angle" to="/$(arg namespace)/thrusters/left_thrust_angle"/>
    <remap from="right_thrust_angle" to="/$(arg namespace)/thrusters/right_thrust_angle"/>
    <remap from="left_thrust_cmd" to="/$(arg namespace)/thrusters/left_thrust_cmd"/>
    <remap from="right_thrust_cmd" to="/$(arg namespace)/thrusters/right_thrust_cmd"/>
  </node>
</launch>
